cmd/podman/import.go

```diff
@@ -107,7 +107,7 @@ func importCmd(c *cli.Context) error {
 		source = file
 	}
 
-	newImage, err := runtime.Import(source, reference, writer, image.SigningOptions{}, config)
+	newImage, err := image.Import(source, reference, writer, image.SigningOptions{}, config, runtime.ImageRuntime())
```

Comment:
This should probably be an ImageRuntime method instead of taking an ImageRuntime as a parameter

Reply:
fixed